### PR TITLE
DOC: update docs and readme

### PR DESCRIPTION
**Pull request by lucyleeow**
Update readme

Including screenshots in using ramp. 

Fix typos.

### CODE REVIEW

**Comment by agramfort** [comment]
nice work @lucyleeow !

can you share the rendered doc links to see how it looks?

**Comment by kegl** [comment]
Beautiful, thanks a lot @lucyleeow!!

**Comment by lucyleeow** [comment]
@agramfort i've amended the CircleCI config to save the built docs as an artifact. The doc is [here](https://13-91170031-gh.circle-artifacts.com/0/home/circleci/project/doc/_build/html/index.html).

File: doc/scoring.rst [hunk 1]
```diff
@@ -61,48 +61,52 @@ look like this::
...
+sample was present in only one CV fold, there is only one prediction for that
+sample. The combined prediction for this sample will simply be that single
+prediction.
 
 Note that technically this is not what bagging means, but the name is used for
 historical reasons.
 
-RAMP event submissions
-======================
+RAMP Studio submissions
```
Comment (agramfort) [inline doc/scoring.rst]: ramp.studio is one instance of the RAMP app. I would not suggest it's **the** only one.

File: doc/using_kits.rst [hunk 2]
```diff
@@ -36,7 +36,7 @@ To get started working on an existing RAMP challenge:
      .. code-block:: bash
 
         $ pip install -r requirements.txt
-        # install ramp-workflow
+        # install ramp workflow
```
Comment (agramfort) [inline doc/using_kits.rst]: the name on pypi etc  would have a - so I would keep it as the name of the package

File: doc/using_kits.rst [hunk 3]
```diff
@@ -48,13 +48,15 @@ To get started working on an existing RAMP challenge:
...
+
     assert_submission()    
 
    See the ``assert_submission()`` `source code
    <https://github.com/paris-saclay-cds/ramp-workflow/blob/master/rampwf/utils/testing.py#L63>`_
    for more details.
 
-Now you are ready to write your own solution for the prediction problem.
+Now you are ready to write your own solution for the prediction problem. It is
+useful to read the starting kit notebook which will outline the
```
Comment (agramfort) [inline doc/using_kits.rst]: missing end of sentence

**Comment by albertcthomas** [comment]
This is great! Thanks @lucyleeow!

**Pull request merged**